bin/cli.js

```diff
@@ -71,7 +71,12 @@ For more information, see https://webpack.js.org/api/cli/.`);
 			options = require("./utils/convert-argv")(argv);
 		} catch (err) {
 			if (err.name !== "ValidationError") {
-				throw err;
+				console.error("\n\u001b[1m\u001b[31mWebpack not found, please install webpack");
```

Comment:
```suggestion
				console.error("\n\u001b[1m\u001b[31mwebpack not found, please install webpack");
```
as per branding guidelines of webpack

Reply:
Thanks @rishabh3112. Will make the following changes.
Can you please suggest some better message for this error?

Reply:
Sure you can,
```suggestion
				console.error("\n\u001b[1m\u001b[31mwebpack not found, please install webpack using\n\tnpm install --save-dev webpack");
```
P.S.
I feel that not having `ValidationError` doesn't strictly implies `webpack` is not installed.
You should check the error's corresponding `err.name` for better handling of this error.

Reply:
@rishabh3112 For this scenario the `err.code` is `MODULE_NOT_FOUND`
IMO I should define a new scope(condition) for handling this particular err.

What do you think about this approach?

Thanks

Reply:
Yeah, you got my point now :), something like below would work IMO:
```suggestion
			if (err.name === "MODULE_NOT_FOUND" && err.message.indexOf("webpack") !== -1) {
      				console.error("\n\u001b[1m\u001b[31m\n\u001b[1m\u001b[31mwebpack not found, please install webpack using\n\tnpm install --save-dev webpack");
			}
```